2024-04-06 00:23:04 -0700 - dagster - DEBUG - __ASSET_JOB - 5ae583e3-4f81-4853-82db-00767e34679d - 3735317 - LOGS_CAPTURED - Started capturing logs in process (pid: 3735317).
2024-04-06 00:23:04 -0700 - dagster - DEBUG - __ASSET_JOB - 5ae583e3-4f81-4853-82db-00767e34679d - 3735317 - reddit_submissions - STEP_START - Started execution of step "reddit_submissions".
2024-04-06 00:23:06 -0700 - dagster - DEBUG - __ASSET_JOB - 5ae583e3-4f81-4853-82db-00767e34679d - 3735317 - reddit_submissions - STEP_OUTPUT - Yielded output "result" of type "Any". (Type check passed).
2024-04-06 00:23:06 -0700 - dagster - DEBUG - __ASSET_JOB - 5ae583e3-4f81-4853-82db-00767e34679d - reddit_submissions - Writing file at: /home/anon/ubuntu-repos/third-times-the-charm/anarchists-cookbook/tmpiilpgri2/storage/reddit_submissions using PickledObjectFilesystemIOManager...
2024-04-06 00:23:06 -0700 - dagster - ERROR - __ASSET_JOB - 5ae583e3-4f81-4853-82db-00767e34679d - 3735317 - reddit_submissions - STEP_FAILURE - Execution of step "reddit_submissions" failed.

dagster._core.errors.DagsterExecutionHandleOutputError: Error occurred while handling output "result" of step "reddit_submissions"::

TypeError: cannot pickle 'duckdb.duckdb.DuckDBPyConnection' object

Stack Trace:
  File "/home/anon/ubuntu-repos/third-times-the-charm/.venv/lib/python3.12/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/home/anon/ubuntu-repos/third-times-the-charm/.venv/lib/python3.12/site-packages/dagster/_utils/__init__.py", line 463, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/home/anon/ubuntu-repos/third-times-the-charm/.venv/lib/python3.12/site-packages/dagster/_core/execution/plan/execute_step.py", line 797, in _gen_fn
    gen_output = output_manager.handle_output(output_context, output.value)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anon/ubuntu-repos/third-times-the-charm/.venv/lib/python3.12/site-packages/dagster/_core/storage/upath_io_manager.py", line 457, in handle_output
    self.dump_to_path(context=context, obj=obj, path=path)
  File "/home/anon/ubuntu-repos/third-times-the-charm/.venv/lib/python3.12/site-packages/dagster/_core/storage/fs_io_manager.py", line 260, in dump_to_path
    pickle.dump(obj, file, PICKLE_PROTOCOL)

